docs/doc_start/15_basics.rst

```diff
@@ -158,7 +158,7 @@ Please refer the :mod:`bones API reference <core.bones>` for all provided classe
 Prototypes and modules
 ======================
 
-Modules are the controllers of a ViUR application, and implement the application logic. To implement modules, ViUR provides four basic prototypes. These are :class:`List<core.prototypes.list.List>`, :class:`Singleton<core.prototypes.singleton.Singleton>`, :class:`Hierarchy<core.prototypes.hierarchy.Hierarchy>` and :class:`Tree<core.prototypes.tree.Tree>`.
+Modules are the controllers of a ViUR application, and implement the application logic. To implement modules, ViUR provides three basic prototypes. These are :class:`List<core.prototypes.list.List>`, :class:`Singleton<core.prototypes.singleton.Singleton>` and :class:`Tree<core.prototypes.tree.Tree>`.
```

Comment:
It is true that we no longer have hierarchy. But for this we have now the `SkelModule`.

```suggestion
Modules are the controllers of a ViUR application, and implement the application logic. To implement modules, ViUR provides four basic prototypes. These are :class:`List<core.prototypes.list.List>`, 
:class:`Singleton<core.prototypes.singleton.Singleton>`,
:class:`Singleton<core.prototypes.skelmodule.SkelModule> and :class:`Tree<core.prototypes.tree.Tree>`.
```